docs/contributors.md

```diff
@@ -105,3 +105,51 @@ First clone and build image as explained in [Commands to build & publish multicl
 helm package charts/multicluster-controlplane --version v0.2.0-kf-alpha.1
 helm push *.tgz oci://quay.io/kubestellar
 ```
+
+### Hosting a copy of postgresql chart and image on quay.io
+
+1. Select the desired postgresql chart tag:
+
+   For example `CHART_TAG=13.1.5`.
+
+2. Pull the chart from the bitmani Docker registry:
+
+   ```shell
+   helm pull oci://registry-1.docker.io/bitnamicharts/postgresql:$CHART_TAG
+   ```
+
+3. Unpack the downloaded `postgresql-$CHART_TAG.tgz` archive:
+
+   ```shell
+   tar xf postgresql-$CHART_TAG.tgz
+   ```
+
+4. In the newly created `postgresql` folder open the `values.yaml` and find the `image:` section:
+
+   - Change the `registry:` from `docker.io` to `quay.io`.
+   - Change the `repository:` from `bitnami/postgresql` to `kubestellar/postgresql`.
+   - Take note of the image tag, for example `IMAGE_TAG=16.0.0-debian-11-r13`.
+
```

Comment:
I would add one more step, for the sake of the cleanliness of the tarchive to be produced: remove any backup files created in the editing steps above.